components/breadcrumb/src/Crumb.js

```diff
@@ -14,38 +14,39 @@ import { SrOnly } from '../../accessibility-helpers/src';
  * Breadcrumb: Breadcrumbs are styled navigational links used to indicate a user journey or path. They are a simple, effective and proven method to aid orientation.
  */
 export const Crumb = ({ children, last, ...props }) => {
-	const { breadcrumb } = useTheme();
-
-	const common = {
-		display: 'inline-block',
-		position: 'relative',
-		color: breadcrumb.color,
-
-		a: {
-			color: breadcrumb.color,
-			textDecoration: 'none',
-
-			'&:focus, &:hover': {
-				textDecoration: 'underline',
-			},
-		},
-	};
+	const { COLORS } = useTheme();
 
 	return (
-		<li css={common} {...props}>
+		<li
+			css={{
+				display: 'inline-block',
+				position: 'relative',
+				color: COLORS.text,
+
+				a: {
+					color: COLORS.text,
+					textDecoration: 'none',
+
+					'&:focus, &:hover': {
+						textDecoration: 'underline',
+					},
+				},
+			}}
+			{...props}
+		>
 			{last && <SrOnly>Current page:</SrOnly>}
 			{children}
-			{!last && (
-				<ArrowRightIcon
-					aria-hidden="true"
-					size="small"
-					color={breadcrumb.separator.color}
-					css={{
-						marginLeft: breadcrumb.separator.marginLeft,
-						marginRight: breadcrumb.separator.marginRight,
-					}}
-				/>
-			)}
+			{!last && '>'
+			// <ArrowRightIcon
+			// 	aria-hidden="true"
+			// 	size="small"
+			// 	color={COLORS.primary}
+			// 	css={{
+			// 		marginLeft: '0.1875rem',
+			// 		marginRight: '0.1875rem',
+			// 	}}
+			// />
+			}
```

Comment:
You can get this working (to test) by fixing the breakpoints desctructure in Icon. There's already a PR for this https://github.com/WestpacGEL/GEL/pull/136

```
...
LAYOUT: { breakpoints },
...
```